lib/suspenders/actions.rb

```diff
@@ -15,7 +15,7 @@ def action_mailer_host(rails_env, host)
     end
 
     def action_mailer_asset_host(rails_env, host)
-      config = "config.action_mailer.asset_host = { host: #{host} }"
+      config = "config.action_mailer.asset_host = #{host}"
```

Comment:
Does the `host` here pass quotes around itself? I think I'd expect something like:

```ruby
config = %{config.action_mailer.asset_host = "#{host}"}
```